staging/velero/README.md

```diff
@@ -0,0 +1,107 @@
+`configuration.extraEnvVars` | Key/values for extra environment variables such as AWS_CLUSTER_NAME, etc | `{}`
+`configuration.provider` | The name of the cloud provider where you are deploying velero to (`aws`, `azure`, `gcp`) |
+`configuration.restoreResourcePriorities` | An ordered list that describes the order in which Kubernetes resource objects should be restored | (uses `velero server` default)
+`configuration.resticTimeout` | How long backups/restores of pod volumes should be allowed to run before timing out. | (uses `velero server` default)
+`configuration.restoreOnlyMode` | When RestoreOnly mode is on, functionality for backups, schedules, and expired backup deletion is turned off. Restores are made from existing backup files in object storage | (uses `velero server` default)
+`configuration.volumeSnapshotLocation.name` | The name of the cloud provider the cluster is using for persistent volumes, if any | `{}`
+`configuration.volumeSnapshotLocation.config.region` | The cloud provider region (AWS only) | ``
+`configuration.volumeSnapshotLocation.config.apitimeout` | The API timeout (`azure` only) |
+`configuration.volumeSnapshotLocation.config.resourceGroup` | The name of the resource group where volume snapshots should be stored, if different from the cluster’s resource group. (Azure only) |
+`configuration.volumeSnapshotLocation.config.project` | The project ID where snapshots should be stored, if different than the project that your IAM account is in. (GCP only) |
+`configuration.volumeSnapshotLocation.config.snapshotLocation` | The location where the snapshots will be stored. (GCP only) |
+`credentials.existingSecret` | If specified and `useSecret` is `true`, uses an existing secret with this name instead of creating one | ``
+`credentials.useSecret` | Whether a secret should be used. Set this to `false` when using `kube2iam` | `true`
+`credentials.secretContents` | Contents for the credentials secret | `{}`
+`snapshotsEnabled` | If `true`, create volumesnapshotlocation crd. Set this to `false` to disable snapshot feature | `true`
+`deployRestic` | If `true`, enable restic deployment | `false`
+`metrics.enabled` | Set this to `true` to enable exporting Prometheus monitoring metrics | `false`
+`metrics.scrapeInterval` | Scrape interval for the Prometheus ServiceMonitor | `30s`
+`metrics.serviceMonitor.enabled` | Set this to `true` to create ServiceMonitor for Prometheus operator | `false`
+`metrics.serviceMonitor.additionalLabels` | Additional labels that can be used so ServiceMonitor will be discovered by Prometheus | `{}`
+`schedules` | A dict of schedules | `{}`
+`restic.podVolumePath` | Location of pod volumes on the host | `/var/lib/kubelet/pods`
+`restic.privileged` | Whether restic should run as a privileged pod. Only necessary in special cases (SELinux) | `false`
+`restic.resources` | Restic DaemonSet resource requests and limits | `{}`
+`configMaps` | Velero ConfigMaps | `[]`
+`minioBackend` | Enable Minio Backend | `false`
```

Comment:
should be placed in alphabetical order? hard to tell here can you check?

Reply:
Again, this is how upstream had it. Need to touch as little as possible.

Reply:
I've split the commits to better show the old vs new, let me know if this is still an issue.